app/src/androidTest/java/org/simple/clinic/DatabaseMigrationAndroidTest.kt

```diff
@@ -861,11 +861,481 @@ class DatabaseMigrationAndroidTest {
+    val columnCount = 12
+
+    val db_v28 = helper.createDatabase(version = 28)
+    db_v28.assertColumnCount(tableName = tableName, expectedCount = columnCount)
+
+    db_v28.execSQL("""
+      INSERT INTO "Appointment" VALUES (
+        'uuid',
+        'patientUuid',
+        'facility-uuid',
+        'scheduled-date',
+        'status',
+        'reason',
+        'remind-on',
+        1,
+        'IN_FLIGHT',
+        'created-at',
+        'updated-at',
+        'deleted-at'
+      )
+    """)
+
+    db_v28.execSQL("""
+      INSERT INTO "Appointment" VALUES (
+        'uuid2',
+        'patientUuid',
+        'facility-uuid',
+        'scheduled-date',
+        'status',
+        'reason',
+        'remind-on',
+        1,
+        'DONE',
+        'created-at',
+        'updated-at',
+        'deleted-at'
+      )
+    """)
+
+    val db_v29 = helper.migrateTo(29)
+    db_v29.query("""SELECT * FROM $tableName""").use {
+      assertThat(it.columnCount).isEqualTo(columnCount)
+
+      it.moveToNext()
+
+      assertThat(it.string("syncStatus")).isEqualTo("PENDING")
+
+      it.moveToNext()
+
+      assertThat(it.string("syncStatus")).isEqualTo("DONE")
+    }
+  }
+
+  @Test
+  fun migration_28_to_29_verify_syncStatus_updated_for_Communication() {
+    val tableName = "Communication"
+    val columnCount = 9
+
+    val db_v28 = helper.createDatabase(version = 28)
+    db_v28.assertColumnCount(tableName = tableName, expectedCount = columnCount)
+
+    db_v28.execSQL("""
+      INSERT INTO "Communication" VALUES (
+        'uuid',
+        'c6834f82-3305-4144-9dc8-5f77c908ebf1',
+        'c64f76b5-0d37-46e2-9426-554e4f809498',
+        'MANUAL_CALL',
+        'AGREED_TO_VISIT',
+        'IN_FLIGHT',
+        '2018-06-21T10:15:58.666Z',
+        '2018-06-21T10:15:58.666Z',
+        '2018-06-21T10:15:58.666Z');
+    """)
+
+    db_v28.execSQL("""
+      INSERT INTO "Communication" VALUES (
+        'uuid2',
+        'c6834f82-3305-4144-9dc8-5f77c908ebf1',
+        'c64f76b5-0d37-46e2-9426-554e4f809498',
+        'MANUAL_CALL',
+        'AGREED_TO_VISIT',
+        'DONE',
+        '2018-06-21T10:15:58.666Z',
+        '2018-06-21T10:15:58.666Z',
+        '2018-06-21T10:15:58.666Z');
+    """)
+
+    val db_v29 = helper.migrateTo(29)
+    db_v29.query("""SELECT * FROM $tableName""").use {
+      assertThat(it.columnCount).isEqualTo(columnCount)
+
+      it.moveToNext()
+
+      assertThat(it.string("syncStatus")).isEqualTo("PENDING")
+
+      it.moveToNext()
+
+      assertThat(it.string("syncStatus")).isEqualTo("DONE")
+    }
+  }
+
+  @Test
+  fun migration_28_to_29_verify_syncStatus_updated_for_MedicalHistory() {
+    val tableName = "MedicalHistory"
+    val columnCount = 12
+
+    val db_v28 = helper.createDatabase(version = 28)
+    db_v28.assertColumnCount(tableName = tableName, expectedCount = columnCount)
+
+    db_v28.execSQL("""
+      INSERT INTO "MedicalHistory" VALUES(
+        'uuid',
+        'patientUuid',
+        0,
```

Comment:
We are using an enum `Answer` now instead of booleans. I assume this is still working because it is inserted as a string, but can you please change this to use the enum values to retain consistency between the database version and the schema?